apollo-gradle-plugin/build.gradle

```diff
@@ -15,6 +15,7 @@ dependencies {
 
   implementation localGroovy()
   implementation project(':apollo-compiler')
+  implementation dep.guavaJre
```

Comment:
Android was pulling guava too and we were using it. Now, it is explicitly defined.